content/YubiHSM2/Usage_Guides/Admin_guide.adoc

```diff
@@ -87,7 +87,7 @@ yubihsm> session open 1 password
 Created session 0
 
 # Create an Authentication Key for Auditing
-yubihsm> put authkey 0 0 "Audit auth key" all audit none $AUDIT_PASS
+yubihsm> put authkey 0 0 "Audit auth key" all get-log-entries none $AUDIT_PASS
```

Comment:
should an "Audit auth key" not be allowed to do `set log index` as well? otherwise there will be problems in `force-audit` mode` (line 94)

Reply:
or is that allowed to be done by all authkeys that have the `get-log-entries` capability?

Reply:
tested `audit set <index>` with the authkey created above and it works.

Why was capability `0x0000000001000000` renamed to `get-log-entries` when it controls both `audit get` and `audit set` commands?

Reply:
Despite being two different commands, setting the log index and extracting entries belong to the same logical operation of getting log entries and, as you noticed.

Reply:
@klali do you know why the quite good and explaining capability name `audit` was changed to a name that not really reflect the full "potential/authority" of the capability.

Reply:
Personally I would have considered changing back to `audit` in yubihsm-shell instead of updating the docs.